services/get-message.yml

```diff
@@ -0,0 +1,27 @@
+- step: get_message
```

Comment:
As discussed internalli, let's use `get_message:` instead of `- step: get_message` to define steps